packages/crew_mvdir.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Crew_mvdir < Package
+  description 'Alternative for "rsync --remove-source-files dir1/ dir2/", with faster speed'
+  homepage 'https://chromebrew.github.io'
+  version '0.1'
+  compatibility 'all'
+  license 'GPL-3'
+
+  source_url 'https://github.com/supechicken/chromebrew/raw/use_rename/src/mvdir.c'
```

Comment:
Should this be versioned and use `https://github.com/chromebrew` repo?

Reply:
Now it is pointed to crew repo, however I am still using `master` since we cannot get the commit hash before this pr merge